Provis.Core/Helpers/StorageTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Provis.Core.Helpers
+{
+    public enum StorageTypes
+    {
+        AzureBlob,
+        Locale
```

Comment:
Do we have an option to save file like a link to google.drive for example 
And should we indicate it ?